lego/apps/meetings/models.py

```diff
@@ -39,6 +42,19 @@ class Meeting(BasisModel):
         related_name="meeting_invitation",
         through_fields=("meeting", "user"),
     )
+    recurring = models.IntegerField(default=-1, null=False, blank=True)
```

Comment:
Very nice job but I would prefer a charField here with predefined choices (recuring and non-recuring). This would mean that you need to change your implementation from a parent creating multiple children, to a parent creating one child, removing itself, and making the new child to a parent (parent meaning recuring)